packages/jellyfish-transaction/src/script/dftx/dftx_balance.ts

```diff
@@ -38,3 +38,24 @@ export class CScriptBalances extends ComposableBuffer<ScriptBalances> {
     ]
   }
 }
+
+/**
+ * TokenBalanceVarInt
+ */
+export interface TokenBalanceVarInt {
+  token: number // ---------------------| VarUInt{1-9 bytes}
+  amount: BigNumber // -----------------| 8 bytes unsigned
+}
```

Comment:
we ady have this 
see https://github.com/DeFiCh/jellyfish/blob/106f44c5997e33c66249a06bd13666ec8f2a6f54/packages/jellyfish-transaction/src/script/dftx/dftx_balance.ts#L6-L9

Reply:
yeah. inconsistencies at c++ side 🥲  It's not the same.
`DCT_ID` in defid is always `VARINT`
refer https://github.com/DeFiCh/ain/blob/6059de0c533710e7f0214128719201a91230bba1/src/amount.h#L73
let me know if you need more input.
@fuxingloh wanted to highlight this.

Reply:
yea that's right